docs/elements.md

```diff
@@ -1032,33 +1032,33 @@ def generate(data):
+| `display`                    | `"block"` or `"inline"` | `"inline"`              | How to display the input field.                                                                                                                                                                                                                                                                                                 |
+| `variables`                  | string                  | —                       | A comma-delimited list of symbols that can be used in the symbolic expression.                                                                                                                                                                                                                                                  |
+| `allow-complex`              | boolean                 | false                   | Whether complex numbers (expressions with `i` or `j` as the imaginary unit) are allowed.                                                                                                                                                                                                                                        |
+| `imaginary-unit-for-display` | string                  | `"i"`                   | The imaginary unit that is used for display. It must be either `"i"` or `"j"`. Again, this is _only_ for display. Both `i` and `j` can be used by the student in their submitted answer, when `allow-complex="true"`.                                                                                                           |
+| `allow-trig-functions`       | boolean                 | true                    | Whether trigonometric functions (`cos`, `atanh`, ...) are allowed.                                                                                                                                                                                                                                                              |
+| `allow-blank`                | boolean                 | false                   | Whether or not an empty input box is allowed. By default, an empty input box will not be graded (invalid format).                                                                                                                                                                                                               |
+| `blank-value`                | string                  | 0 (zero)                | Expression to be used as an answer if the answer is left blank. Only applied if `allow-blank` is `"true"`. Must follow the same format as an expected user input (e.g., same variables, etc.).                                                                                                                                  |
```

Comment:
I believe we're keeping true without quotes, right? I may be wrong, just checking.